src/assets/scripts/client/airport/MapLayerModel.js

```diff
@@ -0,0 +1,118 @@
+import _forEach from 'lodash/forEach';
+import _isEqual from 'lodash/isEqual';
+import _isNumber from 'lodash/isNumber';
+import _map from 'lodash/map';
+import BaseModel from '../base/BaseModel';
+import DynamicPositionModel from '../base/DynamicPositionModel';
+import StaticPositionModel from '../base/StaticPositionModel';
+
+/**
+ * A collection of lines that represent a map layer.
+ *
+ * @class MapLayerModel
+ */
+export default class MapLayerModel extends BaseModel {
+    /**
+     * @for MapLayerModel
+     * @constructor
+     * @param map {object}
+     * @param airportPosition {StaticPositionModel}
+     * @param magneticNorth {number}
+     */
+    constructor(map, airportPosition, magneticNorth) {
+        super();
+
+        if (!map || !airportPosition || !_isNumber(magneticNorth)) {
+            // eslint-disable-next-line max-len
+            throw new TypeError('Invalid parameter, expected map, airportPosition and magneticNorth to be defined');
+        }
+
+        /**
+         * A flag indicating whether the layer should be hidden
+         *
+         * @for MapLayerModel
+         * @property hidden
+         * @type {boolean}
+         * @default false
+         */
+        this.hidden = false;
```

Comment:
booleans should be prefixed with `is` or `has`.  so, in this case: `this.hidden` -> `this.isHidden`

Reply:
I hadn't spotted that convention, the question is, should the json also use isHidden?

Reply:
Yep, they should match each other directly.